Pseudo - Copyedits (#212)
mlsa-book/MLSA: book/P4C21_pseudo.qmd

$$ {#eq-pseudo-value-expectation-1}

This approximate unbiasedness motivates regressing $\tildey_i$ on $\xx_i$ in this reduction.
The relationship between features $\xx_i$ and pseudo-values $\tildey_i(\tau)$ can then be learned through regression, with a separate model fitted for a given time point of interest $\tau$,

$$
\EE[\tildey_i(\tau)\mid\xx_i] = g(\xx_i\mid\bstheta_\tau),
$$ {#eq-pseudo-value-expectation-2}

where $g(\xx_i\mid \bstheta_\tau)$ is the model's output for time point $\tau$, learned by optimizing the parameters $\bstheta_\tau$, for example, $g(\xx_i\mid \bstheta_\tau) = \xx_i^\trans\bstheta_\tau$ for linear regression.\index{linear regression}
The parameters $\bstheta_\tau$, and hence the function, differ for each time point $\tau$.

It follows from (@eq-pseudo-value-expectation-1) and (@eq-pseudo-value-expectation-2) that the fitted regression models can be used to make predictions of the quantity of interest conditional on covariates,

$$
\hat{\psi}(\tau\mid\xx_i) \approx \hat{\tilde{y}}_i = g(\xx_i\mid\hat{\bstheta}_\tau).
$$

Going forward, we will simplify this notation and directly write 
$$\hat{\psi}(\tau \mid \xx_i) = g(\xx_i\mid\hat{\bstheta}_\tau)$$

> when referring to the fitted model.

Author: ```suggestion
Going forward, when referring to the fitted model, this notation is simplified to:

$$
\hat{\psi}(\tau \mid \xx_i) = g(\xx_i\mid\hat{\bstheta}_\tau).
$$
```

> Sometimes, rather than learning $g(\cdot)$ directly, the model learns a *predictor function* $\eta(\xx_i\mid\bstheta_\tau)$, combined with a transformation via a known *response function* $r(\cdot)$, such that 

Author: ```suggestion
Rather than learning $g(\cdot)$ directly, the model often learns a *predictor function* $\eta(\xx_i\mid\bstheta_\tau)$, combined with a transformation via a known *response function* $r(\cdot)$, such that 
```

$$ {#eq-pseudo-value-expectation-2}

where $g(\xx_i\mid \bstheta_\tau)$ is the model's output for time point $\tau$, learned by optimizing the parameters $\bstheta_\tau$, for example, $g(\xx_i\mid \bstheta_\tau) = \xx_i^\trans\bstheta_\tau$ for linear regression.\index{linear regression}
The parameters $\bstheta_\tau$, and hence the function, differ for each time point $\tau$.

It follows from (@eq-pseudo-value-expectation-1) and (@eq-pseudo-value-expectation-2) that the fitted regression models can be used to make predictions of the quantity of interest conditional on covariates,

$$
\hat{\psi}(\tau\mid\xx_i) \approx \hat{\tilde{y}}_i = g(\xx_i\mid\hat{\bstheta}_\tau).
$$

Going forward, we will simplify this notation and directly write 
$$\hat{\psi}(\tau \mid \xx_i) = g(\xx_i\mid\hat{\bstheta}_\tau)$$
when referring to the fitted model.

Sometimes, rather than learning $g(\cdot)$ directly, the model learns a *predictor function* $\eta(\xx_i\mid\bstheta_\tau)$, combined with a transformation via a known *response function* $r(\cdot)$, such that 

$$
g(\xx_i\mid\bstheta_\tau) = r(\eta(\xx_i\mid\bstheta_\tau)).
$$ {#eq-pseudo-value-response-function}

> The response function maps the predictor $\eta$ to the target space (similarly to what was discussed for neural networks in @sec-nnet-activations).

Author: ```suggestion
The response function maps the predictor $\eta$ to the target space (similarly to the discussion in @sec-nnet-activations).
```
